### DISCUSSION TITLE
[Feature] Support for dark mode

### DISCUSSION THREAD

**Discussion by Hephaest** [anuraghazra/github-readme-stats#693] [Ideas]
Is the support for dark mode in the recent roadmap? Well, since Github already launched dark mode, **the single theme cannot be satisfied anymore**.

### idea
The easy way to deal with this issue can be providing two parameters of themes for light and dark mode along with one parameter for mode. The following link from Github can be helpful:

> https://github.community/t/support-theme-context-for-images-in-light-vs-dark-mode/147981

Please check out that. 

Happy to see this feature in the later future. ❤️

**Comment by ghost** [comment]
I agree on this one for sure, but some of the themes are in fact dark

**Comment by 2br-2b** [comment]
Hi, thanks for the feedback! There's more in-depth discussion going on at #690, so maybe you could provide your idea there! Thanks!

**Comment by rickstaa** [comment]
You can also check out https://github.com/anuraghazra/github-readme-stats/discussions/1415#discussioncomment-2152317 on information on how to switch themes based on the Github mode.

**Comment by rickstaa** [comment]
All the currently available ways for creating a dynamic theme are now documented in the [README.md](https://github.com/anuraghazra/github-readme-stats#responsive-card-theme) (See https://github.com/anuraghazra/github-readme-stats#responsive-card-theme).
